### DISCUSSION TITLE
compute_columns3d not implemented for Half

### DISCUSSION THREAD

**Discussion by Rajesh-ParaxialTech** [Project-MONAI/MONAI#8104] [Q&A] [answered]
Hello 

I was trying to train a lung nodule detection model with a small dataset using the command line ---------

python -m monai.bundle run --config_file configs/train.json

and came across the following error log:

![image](https://github.com/user-attachments/assets/22890bca-b288-4364-9bc5-6ec137cbc8da)

If I could get help to overcome this error log, it would be great. Thanking you

Rajesh

**Comment by KumoLiu** [comment, marked as answer]
Hi @Rajesh-ParaxialTech, based on the error message, it appears that the data dtype is incompatible with the network implementation. A possible workaround would be to add an EnsureType transform before passing the data into the network to ensure it's in float32 format.

Hope it helps, thanks.

**Comment by Rajesh-ParaxialTech** [comment]
Thank you YunLiu
